font-types/src/uint24.rs

```diff
@@ -55,6 +55,26 @@ impl From<Uint24> for usize {
     }
 }
 
+#[derive(Debug)]
```

Comment:
All public types should have docs here (we're lax on it elsewhere but we're quite good in font-types)